community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -99,6 +100,7 @@
  */
 public class ImportTool
 {
+    private static final String UNLIMITED = "-";
```

Comment:
I do think -1 isn't particularly user friendly, is it? I'd vote for `true`, as discussed above